src/app/docs/radio/radio-docs.component.html

```diff
@@ -1,13 +1,333 @@
+      >
+        <p class="sky-docs-text-block">
+          Icon radio buttons can use background and border color that match <a href="https://developer.blackbaud.com/skyux/components/status-indicator#options">status indicator types</a> such as success or danger.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior & states"
+    >
+      <table class="sky-docs-table">
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Selected
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/radio-button/radio-button-combined-state-2.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Unselected
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/radio-button/radio-button-combined-state-1.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Hover
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/radio-button/radio-button-combined-state-3.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Disabled
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/radio-button/radio-button-combined-state-4.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+      </table>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <p class="sky-docs-text-block">
+        Radio buttons may or may not use a preceeding label to provide context for the options. Unlike text input labels, this label may use verbs or questions. If you do not include a label, provide context for the consequences of the choices in the radio button text labels.
+      </p>
+
+      <p class="sky-docs-text-block">
+        Icon radio buttons use the <a href="https://fontawesome/v4.7.0/">FontAwesome 4.7</a> icon library. For a list of common icons, see the <a href="https://developer.blackbaud.com/skyux/design/styles/icons">SKY UX icon dictionary</a>. Icon radio buttons must use a <sky-code>label</sky-code> or <sky-code>labelledBy</sky-code> property for accessibility.
```

Comment:
I'm getting a DNS error when I try to access https://fontawesome/v4.7.0/.

Reply:
Fixed. The link was missing ".com"